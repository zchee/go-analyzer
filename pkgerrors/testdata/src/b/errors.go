package a

import (
	"fmt"

	"github.com/pkg/errors" // want `found use "github.com/pkg/errors" package`
)

func PkgErrorsNew() error {
	return errors.New("whoops") // ok
}

func PkgErrorsErrorf() error {
	foo := fmt.Sprintf("foo")
	return errors.Errorf("whoops: %s", foo) // want `found use location of the deprecated github.com/pkg/errors`
}
