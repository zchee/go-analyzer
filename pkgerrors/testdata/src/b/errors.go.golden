package a

import (
	"fmt"

	"errors" // want `found use "github.com/pkg/errors" package`
)

func PkgErrorsNew() error {
	return errors.New("whoops") // ok
}

func PkgErrorsErrorf() error {
	foo := fmt.Sprintf("foo")
	return fmt.Errorf("whoops: %s", foo) // want `found use location of the deprecated github.com/pkg/errors`
}
